kidsability/src/containers/Login.css

```diff
@@ -0,0 +1,10 @@
+@media all and (min-width: 480px) {
+    .Login {
+      padding: 60px 0;
+    }
+  
+    .Login form {
+      margin: 0 auto;
+      max-width: 320px;
+    }
+  }
```

Comment:
I think we should separate the css files from the js files. So maybe create a separate folder called styles and put all the css files there.

Reply:
Sounds good